crates/recorder/src/measure/wall_cycles.rs

```diff
@@ -2,27 +2,34 @@
 //! wrapper around the `precision` crate to adapt it to the [Measure] API.
 
 use super::{Measure, Measurements};
+use lazy_static::lazy_static;
 use precision::{Config, Precision, Timestamp};
 use sightglass_data::Phase;
+use std::ops::Deref;
 
-pub struct WallCycleMeasure(Precision, Option<Timestamp>);
+lazy_static! {
+    static ref PRECISION: Precision = Precision::new(Config::default()).unwrap();
+}
+
+pub struct WallCycleMeasure(Option<Timestamp>);
 
 impl WallCycleMeasure {
     pub fn new() -> Self {
-        let precision = Precision::new(Config::default()).unwrap();
-        Self(precision, None)
+        Self(None)
     }
 }
 
 impl Measure for WallCycleMeasure {
     fn start(&mut self) {
-        self.1 = Some(self.0.now());
+        let start = PRECISION.now();
+        self.0 = Some(start);
     }
 
     fn end(&mut self, phase: Phase, measurements: &mut Measurements) {
-        let end = self.0.now();
-        let elapsed = end - self.1.take().expect("an existing timestamp");
+        let precision = PRECISION.deref();
```

Comment:
Why do we have to `deref()` here in `end` but not in `start`?

Reply:
Ah, so it's implicit in `start()` but I explicitly deref'd once here to avoid doing the lazy-init atomic access and branch twice. Though you have a good point in that the explicit trait call is a bit incongruent; I switched to the usual `&*PRECISION` idiom.